doc/user/content/sql/alter-cluster.md

```diff
@@ -6,44 +6,159 @@ menu:
     parent: 'commands'
 ---
 
-`ALTER CLUSTER` changes the configuration of a cluster, which **might incur
-downtime**. To rename a cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
+`ALTER CLUSTER` changes the configuration of a cluster, such as the `SIZE` or
+`REPLICATON FACTOR`.
 
 ## Syntax
 
-#### Set new configuration value
+`ALTER CLUSTER` has the following syntax forms:
 
-{{< diagram "alter-cluster-set.svg" >}}
+{{< tabs >}}
+{{< tab "Set a configuration" >}}
 
-#### Reset configuration to original value
+To set a cluster configuration:
 
-{{< diagram "alter-cluster-reset.svg" >}}
+  ```mzsql
+  ALTER CLUSTER <cluster_name>
+  SET (
+      SIZE = <text>,
+      REPLICATION FACTOR = <int>,
+      INTROSPECTION INTERVAL = <interval>,
+      INTROSPECTION DEBUGGING = <bool>,
+      MANAGED = <bool>,
+      SCHEDULE = { MANUAL | ON REFRESH (...) }
+  )
+  [WITH ({ WAIT UNTIL READY({TIMEOUT | ON TIMEOUT {COMMIT|ROLLBACK}}) | WAIT FOR <duration> })]
+  ;
+  ```
 
-### Cluster options
+You must have ownership of the cluster. See also [Required
+privileges](#required-privileges).
```

Comment:
Is it helpful to point to the required privileges section if none of the other privileges apply to the syntax in this tab?Users will get an error if they don't have the right privileges. It'd be easier to maintain the privileges section if we just kept it in that one place, but if we want to add something more useful, we could include the commands to verify the current ownership and add some more practical advice (e.g., altering ownership of the cluster to a role the requesting role is also a member of).

Reply:
Can remove.